### PR TITLE
Gumchu girl

### PR DESCRIPTION
**Pull request by MadShmupper**
# New Port for { GUMCHU GIRL }

## Game Information
- **Title**: { GUMCHU GIRL }
- **URL**: { https://caseportman.itch.io/gumchu-girl-demo }

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [x] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/simonsqrev/README.md
Comment (kloptops) [inline ports/simonsqrev/README.md:1]: Why is this port being affected here? 🐢

**Review by kloptops — CHANGES_REQUESTED** [review]

File: ports/simonsqrev/README.md
Comment (MadShmupper) [reply ports/simonsqrev/README.md:1]: Yikes. I'm sorry. I'm uncertain what I may have done. I'll look closer when I wake.  Sorry

Comment (Cebion) [reply ports/simonsqrev/README.md:1]: This happens if you don't switch to main before making a new branch for example =)

**Comment by MadShmupper** [comment]
Closing to fix it. Will reopen soon

**Pull request closed without merging**